.github/workflows/linux_conditional_compilation.yml

```diff
@@ -207,14 +213,14 @@ jobs:
         - /mount/caches:/mount/caches
```

Comment:
looks like you need to pass `options` to container, because sscache stat shows 0 hit rate

Reply:
Fixed it